unblob/strategies.py

```diff
@@ -38,9 +40,15 @@ def search_chunks_by_priority(path: Path, file: io.BufferedReader) -> List[Chunk
                 real_offset = offset + handler.YARA_MATCH_OFFSET
                 limited_reader = LimitedStartReader(file, real_offset)
                 chunk = handler.calculate_chunk(limited_reader, real_offset)
+
                 # We found some random bytes this handler couldn't parse
                 if chunk is None:
                     continue
+
+                if chunk.end_offset >= file_size:
```

Comment:
Would something like this make sense?

```suggestion
                if chunk.end_offset >= file_size or chunk_size.start_offset < 0:
```

Reply:
We fixed this by using the 2nd diff